api/types/constants.go

```diff
@@ -326,6 +326,9 @@ const (
 	// KindPlugin represents a plugin instance
 	KindPlugin = "plugin"
 
+	// KindPluginStaticCredentials represents plugin static credentials.
+	KindPluginStaticCredentials = "plugin_static_credentials"
```

Comment:
Nit: Does there need to be the word "static" in the name?

Reply:
There doesn't, but I'm worried about the confusion here since there's an existing plugin field called `PluginCredentials`. Thoughts?

Reply:
Ah I see. I think it's fine then.